tests/tacacs/test_authorization.py

```diff
@@ -0,0 +1,342 @@
+import crypt
+import paramiko
+import pytest
+
+from .test_ro_user import ssh_remote_run
+from tests.common.helpers.assertions import pytest_assert
+from .utils import stop_tacacs_server, start_tacacs_server
+
+logger = logging.getLogger(__name__)
+
+TIMEOUT_LIMIT   = 120
+
+pytestmark = [
+    pytest.mark.disable_loganalyzer,
+    pytest.mark.topology('any'),
+    pytest.mark.device_type('vs')
+]
+
+def setup_local_user(duthost, creds_all_duts):
+    try:
+        duthost.shell("sudo deluser {}".format(creds_all_duts[duthost]['tacacs_local_user']))
+    except:
+        logger.warning("local user not exist")
+    
+    duthost.shell("sudo useradd {}".format(creds_all_duts[duthost]['tacacs_local_user']))
+    duthost.shell('sudo echo "{}:{}" | chpasswd'.format(creds_all_duts[duthost]['tacacs_local_user'],creds_all_duts[duthost]['tacacs_local_user_passwd']))
+
+def ssh_connect_remote(remote_ip, remote_username, remote_password):
+    ssh = paramiko.SSHClient()
+    ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+    ssh.connect(remote_ip, username=remote_username, password=remote_password, allow_agent=False, look_for_keys=False, auth_timeout=TIMEOUT_LIMIT)
+    return ssh
+    
+
+def check_ssh_connect_remote_failed(remote_ip, remote_username, remote_password):
+    login_failed = False
+    try:
+        ssh_client_local = ssh_connect_remote(remote_ip, remote_username, remote_password)
+    except paramiko.ssh_exception.AuthenticationException as e:
+        login_failed = True
+    
+    pytest_assert(login_failed == True)
+
+def ssh_run_command(ssh_client, command):
```

Comment:
>ssh_run_command [](http://example.com/codeflow?start=4&length=15)

Better to check return code of command first. this function `recv_exit_status` my help.

Reply:
Fixed, also add check for return value.